tests/mocks.py

```diff
@@ -145,6 +147,19 @@ def revision(
             }
         )
 
+        next_free_txn_id = len(self._transactions)
+        for txn_id, comment in enumerate(comments, start=next_free_txn_id):
+            # See https://phabricator.services.mozilla.com/conduit/method/transaction.search/  # noqa
+            # for the Conduit API result object structure.
+            self._transactions.append(
+                {
+                    "id": txn_id,
```

Comment:
This should use the id generation helper method and transactions (really, every object) needs a phid:
```suggestion
                    "id": self._new_id(self._transactions),
                    "phid": self._new_phid("PHID-XACT-DREV"),
```